components/compliance-service/reporting/relaxting/migrations.go

```diff
@@ -354,6 +383,17 @@ func (backend ES2Backend) getScanDateRange(indexPrefix string) (*time.Time, *tim
 		return nil, nil, errors.Wrap(err, fmt.Sprintf("%s unable to read max_date for range", myName))
 	}
 
+	if minDate.Value == nil {
+		//if we are here, it means that there are indices with this prefix but not any elasticsearch documents within those indices
+		//if there were, then mindDate.value would be non-nil
+		//in this case, we need to remove all of the indices with this prefix.. they are all empty anyway
+		_, err := cleanupEmptyIndicesForPrefix(client, esIndex, migratable)
+		if err != nil {
+			return nil, nil, errors.Wrap(err, fmt.Sprintf("%s unable to delete the empty indices that need to be removed", myName))
+		}
+		return nil, nil, errors.Errorf("%s there were not any reports in elasticsearch indices that are prefixed with %s, these indices, therefore, have been successfully deleted", myName, esIndex)
```

Comment:
Should we return an error if there were no indices to migrate? We could return `nil, nil, nil` and the migration would show a success.

Reply:
yes. thanks, @lancewf.  We no longer return an error here, instead, we log that as a warning and move on